src/sys/eventfd.rs

```diff
@@ -10,8 +10,58 @@ libc_bitflags! {
     }
 }
 
+#[deprecated(since = "0.27.0", note = "Use EventFd::value_and_flags() instead")]
 pub fn eventfd(initval: libc::c_uint, flags: EfdFlags) -> Result<OwnedFd> {
     let res = unsafe { libc::eventfd(initval, flags.bits()) };
 
     Errno::result(res).map(|r| unsafe { OwnedFd::from_raw_fd(r) })
 }
+
+#[derive(Debug)]
+pub struct EventFd(pub OwnedFd);
+impl EventFd {
+    /// [`EventFd::value_and_flags`] with `init_val = 0` and `flags = EfdFlags::empty()`.
+    pub fn new() -> Result<Self> {
+        let res = unsafe { libc::eventfd(0, EfdFlags::empty().bits()) };
+        Errno::result(res).map(|r| Self(unsafe { OwnedFd::from_raw_fd(r) }))
+    }
+    /// Constructs [`EventFd`] with the given `init_val` and `flags`.
+    /// 
+    /// Wrapper around [`libc::eventfd`].
+    pub fn value_and_flags(init_val: u32, flags: EfdFlags) -> Result<Self> {
```

Comment:
It would be more idiomatic to name these constructors from_XXX

Reply:
Renamed with `from_` prefix.